docs/getting-started/new-project-guide/go_lang.md

```diff
@@ -27,10 +27,47 @@ only. In that mode, fuzz targets for Go use the libFuzzer engine with native Go
 coverage instrumentation. Binaries compiled in this mode provide the same
 libFuzzer command line interface as non-Go fuzz targets.
 
+## Native Go 1.18 support
```

Comment:
Hmm...Should we just call this Native Go Fuzzing support or something instead of 1.18?
Will we have to change this doc when 1.19 comes out?